gremlin-go/example/go.sum

```diff
@@ -0,0 +1,22 @@
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
+github.com/google/uuid v1.3.0 h1:t6JiXgmwXMjEs8VusXIJk2BXHsn+wx8BZdTaoZ5fu7I=
+github.com/google/uuid v1.3.0/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
+github.com/gorilla/websocket v1.4.2 h1:+/TMaTYc4QFitKJxsQ7Yye35DkWvkdLcvGKqM+x0Ufc=
+github.com/gorilla/websocket v1.4.2/go.mod h1:YR8l580nyteQvAITg2hZ9XVh4b55+EU/adAjf1fMHhE=
+github.com/lyndonbauto/tinkerpop/gremlin-go/v3 v3.5.3-rc12 h1:62id3eBpn79wef663sf0LHecGbR28vGdKKLAG63kPVk=
+github.com/lyndonbauto/tinkerpop/gremlin-go/v3 v3.5.3-rc12/go.mod h1:3GYWNfYdr+2wpLwsdXUQftwrmhUFA+Vfq3KWXjhY6Fw=
+github.com/nicksnyder/go-i18n/v2 v2.1.2 h1:QHYxcUJnGHBaq7XbvgunmZ2Pn0focXFqTD61CkH146c=
```

Comment:
Needs to be apache/tinkerpop

Reply:
updated in https://github.com/lyndonbauto/tinkerpop/pull/94/commits/e2e86986e299552753eab71559aa9cb7fd18643d